src/DssSpell.t.sol

```diff
@@ -1334,25 +1334,25 @@ contract DssSpellTest is DSTest, DSMath {
         checkCollateralValues(afterSpell);
     }
 
-    function testChainlogValues() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+    // function testChainlogValues() public {
```

Comment:
Not sure if you intentionally left these commented out or not, but to me it makes sense to leave these tests in even if it's essentially a no-op for this particular spell. It's possible someone forgets to re-add them in the future, or they think these are no longer necessary, etc.

Reply:
I intentionally removed them to make the CI run a bit faster.
My reasoning was that at the moment these tests are necessary for e.g. a new chainlog entry, whoever is working on the spell will still need to make addendums to that test in order to test the new entry.
So basically someone forgetting to uncomment these tests would also forget to add the new entries in the actual no-op test.
However if you guys feel is better to keep them uncommented, not strongly opposed either.

Reply:
I agree on commenting `testChainlogValues()` and `testCollateralIntegrations()`.
I would keep the rest uncommented, `test_OSMs()` and `test_Medianizers()`